src/bot/integrations/discord.ts

```diff
@@ -371,6 +385,27 @@ class Discord extends Integration {
     } catch (e) {
       warning(e.stack);
     }
+
+    try {
+      this.changeClientOnlinePresence();
+    } catch (e){
+      warning(e.stack);
+    }
+  }
+
+  @onChange('onlinePresence')
+  async changeClientOnlinePresence() {
+    try {
+      if (api.isStreamOnline) {
+        this.client?.user?.setStatus('online');
+        this.client?.user?.setPresence( { status: 'online', activity: {name: `${api.stats.currentTitle}`, type: 'STREAMING', url: `https://twitch.tv/${oauth.generalChannel}`} } );
```

Comment:
also this seem to be confusing to me as well...we should have configurable online presense for offline/online stream states. Also we should have configurable activity as well (maybe just toggle to set streaming activity during online stream) -> thus keeping old behavior